debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/connection/Lsn.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.postgresql.connection;
+
+import java.nio.ByteBuffer;
+
+import org.postgresql.replication.LogSequenceNumber;
+
+/**
+ * Abstraction of PostgreSQL log sequence number, adapted from
+ * {@link org.postgresql.replication.LogSequenceNumber}.
+ *
+ * @author Jiri Pechanec
+ *
+ */
+public class Lsn implements Comparable<Lsn> {
```

Comment:
Wasn't DBZ-2200 intended to have a more abstract implementation in debezium-core that fits different connector needs?

Reply:
No, at least that's not what I had in mind when logging this. This really was focused on having such `Lsn` type for PG instead of representing it with `Long`.

Reply:
I meant when we were talking about the task. At least that what I understood. Does it makes sense to have it in debezium-core?

Reply:
It doesn't make sense to have `Lsn` in core, as it's PG-specific. I'm not sure whether one could finda more generic abstraction which still would be useful tbh. In any case I don't think it's something we need to concern ourselves with right now. When it comes to (simple-ish) refactorings, I'd suggest rather to look at https://issues.redhat.com/browse/DBZ-2420 (unification of JDBC connection props) or the filter handling unification.